GraphicEditor/View/Windows/EquipmentSearch.xaml.cs

```diff
@@ -23,33 +25,32 @@ private void SearchEquipmentOrMedicine(object sender, RoutedEventArgs e)
         {
             string itemName = EquipmentNameTextBox.Text;
 
-            List<Equipment> equipments = equipmentRepository.GetEquipmentsByName(itemName);
-            List<Medicine> medicines = medicineRepository.GetMedicinesByName(itemName);
+            List<Equipment> equipments = equipmentRepository.GetByName(itemName);
+            List<Medicine> medicines = medicineRepository.GetByName(itemName);
 
             GenerateReport(itemName, equipments, medicines);
             EquipmentNameTextBox.Text = null;
         }
 
-        private void GenerateReport(string itemName, List<Equipment> equipments, List<Medicine> medicines)
+        private void GenerateReport(string itemName, List<Equipment> equipments, List<Medicine> medicine)
```

Comment:
This could also be done in the service and then the view model would get this text as a result and update the property